packages/pwa-kit-dev/src/ssr/server/build-dev-server.js

```diff
@@ -93,17 +92,6 @@ export const DevServerMixin = {
         )
     },
 
-    /**
-     * @private
-     */
-    _setupLogging(app) {
```

Comment:
We need to remove the function in the dev server so the one in remote server can take affect on both